tensorflow/tools/ci_build/install/install_deb_packages.sh

```diff
@@ -43,7 +43,7 @@ apt-get install -y --no-install-recommends \
     autoconf \
     automake \
     build-essential \
-    clang-format-3.8 \
+    clang-format-3.9 \
```

Comment:
Is this package necessary?
I feel like it may be a leftover package.

Reply:
I will try some test on x86 without this package and verify it works ok.